src/sigma/test/serialize_test.cpp

```diff
@@ -19,6 +19,25 @@ BOOST_AUTO_TEST_CASE(group_element_serialize)
     BOOST_CHECK(initial == resulted);
 }
 
+BOOST_AUTO_TEST_CASE(group_element_invalid)
+{
+    secp_primitives::GroupElement initial;
+    initial.randomize();
+    unsigned char buffer [initial.memoryRequired()];
+    initial.serialize(buffer);
+    //Shift all elements to right by 2, to make it invalid
+    for (int i = initial.memoryRequired() - 1; i > 1; i--)
+        buffer[i] = buffer[i - 2];
+    buffer[0] = std::rand();
+    buffer[1] = std::rand();
+
+    // Making the point not infinity as it is allowed in deserialization
+    buffer[33] = 0;
```

Comment:
Are there nonstandard representations of the zero point that need to be checked for validity?

Reply:
I don't think there are such representations.